packages/react-components/src/components/Picker/Picker.tsx

```diff
@@ -32,6 +32,7 @@ export interface IPickerProps {
   type?: PickerType;
   searchDisabled?: boolean;
   onSelect: (selectedItems: IPickerListItem[] | null) => void;
+  dataTestId?: string;
```

Comment:
The current prop name can be confusing, as we can pass the `data-testid` to Picker component.
```suggestion
  triggerDataTestId?: string;
```